pgmpy/readwrite/BIF.py

```diff
@@ -248,15 +239,22 @@ def get_cpd(self):
             name = self.probability_expr.searchString(block)[0][0]
             cpds = self.cpd_expr.searchString(block)
             arr = [float(j) for i in cpds for j in i]
-            arr = numpy.array(arr)
-            arr = arr.reshape((len(self.variable_states[name]),
-                              arr.size//len(self.variable_states[name])))
+            if 'table' in block:
+                arr = numpy.array(arr)
+                arr = arr.reshape((len(self.variable_states[name]),
+                                   arr.size // len(self.variable_states[name])))
+            else:
+                length = len(self.variable_states[name])
+                reshape_arr = [[] for i in range(length)]
+                for i, val in enumerate(arr):
+                    reshape_arr[i % length].append(val)
```

Comment:
I'm manually reshaping the array, because values were given like this
```
() v1 1-v1
() v2 1-v2
() v3 1-v3
```
and were to be converted like
```
[ v1, v2, v3]
[1-v1, 1-v2, 1-v3]
```
Couldn't find another method, for this.